addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1329,6 +1339,20 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
                     || dependencyEl.nodeName === 'TEXTAREA') && !['set', '!set'].includes(this.$target[0].dataset.visibilityComparator);
             case 'hidden_condition_no_text_opt':
                 return dependencyEl && (dependencyEl.type === 'checkbox' || dependencyEl.type === 'radio' || dependencyEl.nodeName === 'SELECT');
+            case "hidden_condition_no_text_only_opt":
+                return (
+                    dependencyEl &&
+                    (dependencyEl.type === "checkbox" ||
+                        dependencyEl.type === "radio" ||
+                        dependencyEl.nodeName === "SELECT") &&
+                    ["selected", "!selected"].includes(this.$target[0].dataset.visibilityComparator));
```

Comment:
- Do it with 2 lines instead of 6. The first line should be exactly the same as the one for `hidden_condition_no_text_opt`.
- No need for the first parenthesis that wrap everything.

Reply:
I guess linter will not be happy if we make it in 2 line

Reply:
> I guess linter will not be happy if we make it in 2 line

Personally, I don't mind this line since the other lines around it are long too. But do what you prefer